modules/videoio/cmake/detect_ffmpeg.cmake

```diff
@@ -51,6 +51,7 @@ if(HAVE_FFMPEG AND NOT HAVE_FFMPEG_WRAPPER)
   set(_min_libavutil_version 52.3.0)
   set(_min_libswscale_version 2.1.1)
   set(_min_libavresample_version 1.0.1)
+  set(_min_libavdevice_version 56.0.0)
```

Comment:
> 56.0.0

Why?

Version should be aligned with other components, e.g. `54.20.0`

Reply:
That was the oldest version I had in my repositories and was able to test. Should i build and test against ```54.20.0```

Reply:
The oldest build configuration which we have is `libav` from Ubuntu 14.04 (see PR #18146):

```
# pkg-config --modversion libavdevice    
53.2.0
```

So please use `53.2.0` here.

Reply:
Alright

Reply:
Done: 907c8840dc63b6e41076f3222256816556d5e47a